www/shared/utils/google-tag-manager/initialize-tag-manager.js

```diff
@@ -0,0 +1,15 @@
+import TagManager from 'react-gtm-module';
+
+const tagManagerInitialArgs = {
+    gtmId: process.env.GTM_TRACKING_ID,
+};
+
+export const initializeTagManager = (args) => {
+    if (!process.env.GTM_TRACKING_ID) {
+        return;
+    }
+
+    TagManager.initialize({ ...tagManagerInitialArgs, ...args });
+};
+
+export default initializeTagManager;
```

Comment:
Do we have named exports and a default export for any particular reason? The module itself is called `initialize-tag-manager`. As such, why do we need to have a named export? Isn't the default one enough?